tika-dl/pom.xml

```diff
@@ -87,6 +87,11 @@
       <artifactId>nd4j-native-platform</artifactId>
       <version>${dl4j.version}</version>
     </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-compress</artifactId>
+        <version>1.14</version>
```

Comment:
fixed in https://github.com/apache/tika/commit/94f8b9fe5fdaebd11a99e76dd742bdc6df427389